include/triton/Dialect/Triton/Transforms/Utility.h

```diff
@@ -0,0 +1,15 @@
+#ifndef TRITON_TRANSFORMS_UTILITY_H
+#define TRITON_TRANSFORMS_UTILITY_H
+
+#include "triton/Dialect/Triton/IR/Dialect.h"
+
+using namespace mlir;
+
+namespace mlir::triton {
+
+Value getPredMask(RewriterBase &rewriter, Type typeLike, Value currentMask,
```

Comment:
should go in include/triton/Dialect/Triton/IR/Utility.h

Reply:
done